kernel/include/paging.h

```diff
@@ -34,23 +40,22 @@ extern uint64_t* page_zone_pointer;
 #define PAGETABLE_DEFAULT (PAGING_PRESENT | PAGING_WRITABLE)
```

Comment:
Is there such a thing as default flags for a page? And if so why is the default writable?